modules/core/shared/src/test/scala/scaladex/core/test/Values.scala

```diff
@@ -159,38 +172,31 @@ object Values {
     val license: License = License("MIT License", "MIT", Some("https://spdx.org/licenses/MIT.html"))
-
-    private def getArtifact(
-        name: String,
-        binaryVersion: BinaryVersion,
-        version: SemanticVersion,
-        description: Option[String] = None,
-        fullScalaVersion: Option[SemanticVersion] = None
-    ): Artifact = {
-      val artifactId = ArtifactId(Name(name), binaryVersion)
-      Artifact(
-        groupId = groupId,
-        artifactId = artifactId.value,
-        version = version,
-        artifactName = artifactId.name,
-        platform = binaryVersion.platform,
-        language = binaryVersion.language,
-        projectRef = reference,
-        description = description,
-        releaseDate = Instant.ofEpochMilli(1620911032000L),
-        resolver = None,
-        licenses = Set(license),
-        isNonStandardLib = false,
-        fullScalaVersion = fullScalaVersion
-      )
-    }
-
     val `core_3:2.6.1`: Artifact = getArtifact(
       "cats-core",
       `_3`,
       `2.6.1`,
       description = Some("Cats core"),
-      fullScalaVersion = SemanticVersion.parse("3.0.0")
+      fullScalaVersion = SemanticVersion.parse("3.0.0"),
+      scaladocUrl = Some(Url("http://typelevel.org/cats/api/")),
+      versionScheme = Some("semver-spec"),
+      developers = Seq(
+        developer("Cody Allen", "https://github.com/ceedubs/", "org.typelevel:cats-core_3:jar:2.6.1"),
+        developer("Ross Baker", "https://github.com/rossabaker/", "org.typelevel:cats-core_3:jar:2.6.1"),
+        developer("P. Oscar Boykin", "https://github.com/johnynek/", "org.typelevel:cats-core_3:jar:2.6.1"),
+        developer("Travis Brown", "https://github.com/travisbrown/", "org.typelevel:cats-core_3:jar:2.6.1"),
+        developer("Adelbert Chang", "https://github.com/adelbertc/", "org.typelevel:cats-core_3:jar:2.6.1"),
+        developer("Peter Neyens", "https://github.com/peterneyens/", "org.typelevel:cats-core_3:jar:2.6.1"),
+        developer("Rob Norris", "https://github.com/tpolecat/", "org.typelevel:cats-core_3:jar:2.6.1"),
+        developer("Erik Osheim", "https://github.com/non/", "org.typelevel:cats-core_3:jar:2.6.1"),
+        developer("LukaJCB", "https://github.com/LukaJCB/", "org.typelevel:cats-core_3:jar:2.6.1"),
+        developer("Michael Pilquist", "https://github.com/mpilquist/", "org.typelevel:cats-core_3:jar:2.6.1"),
+        developer("Miles Sabin", "https://github.com/milessabin/", "org.typelevel:cats-core_3:jar:2.6.1"),
+        developer("Daniel Spiewak", "https://github.com/djspiewak/", "org.typelevel:cats-core_3:jar:2.6.1"),
+        developer("Frank Thomas", "https://github.com/fthomas/", "org.typelevel:cats-core_3:jar:2.6.1"),
+        developer("Julien Truffaut", "https://github.com/julien-truffaut/", "org.typelevel:cats-core_3:jar:2.6.1"),
+        developer("Kailuo Wang", "https://github.com/kailuowang/", "org.typelevel:cats-core_3:jar:2.6.1")
```

Comment:
It's better, but it is still repeated 3 times. Maybe you can introduce this?

```scala
def developers(id: String) = Seq(
        developer("Cody Allen", "https://github.com/ceedubs/", id),
        developer("Ross Baker", "https://github.com/rossabaker/", id),
        developer("P. Oscar Boykin", "https://github.com/johnynek/", id),
        developer("Travis Brown", "https://github.com/travisbrown/", id),
        developer("Adelbert Chang", "https://github.com/adelbertc/", id),
        developer("Peter Neyens", "https://github.com/peterneyens/", id),
        developer("Rob Norris", "https://github.com/tpolecat/", id),
        developer("Erik Osheim", "https://github.com/non/", id),
        developer("LukaJCB", "https://github.com/LukaJCB/", id),
        developer("Michael Pilquist", "https://github.com/mpilquist/", id),
        developer("Miles Sabin", "https://github.com/milessabin/", id),
        developer("Daniel Spiewak", "https://github.com/djspiewak/", id),
        developer("Frank Thomas", "https://github.com/fthomas/", id),
        developer("Julien Truffaut", "https://github.com/julien-truffaut/", id),
        developer("Kailuo Wang", "https://github.com/kailuowang/", id)
)

```

Reply:
fixed it.